### DISCUSSION TITLE
Is there an option to select multiple boxes and move them?

### DISCUSSION THREAD

**Discussion by SelimOzel** [MicroPad/Web#435] [General]
Thanks for creating this. I use it track a few tasks and I love it. The question's is in the title.

Best,
S

**Comment by NickGeek** [comment]
Hi, thanks for the suggestion!

There isn't a way to do that currently, for my tasks I tend to group the text all together and then just cut/paste the text into other elements.

It should be possible to add something like that though, but I don't have the time to do so soon.

**Reply by NickGeek** [reply]
I've raised https://github.com/MicroPad/MicroPad-Core/issues/436 for this feature request.
